src/entity/model.rs

```diff
@@ -2,7 +2,7 @@ use crate::{DbErr, EntityTrait, Linked, QueryFilter, QueryResult, Related, Selec
 pub use sea_query::Value;
 use std::fmt::Debug;
 
-pub trait ModelTrait: Clone + Debug {
```

Comment:
Oh we needed to add `Send` to `ModelTrait`

Reply:
Yes, this this what I'm taking